code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -856,6 +856,8 @@
 
 	product_ads = "If it moves, it's hostile!;How many enemies have you killed today?;Shoot first, perform autopsy later!;Your ammo is right here.;Guns!;Die, scumbag!;Don't shoot me bro!;Shoot them, bro.;Why not have a donut?"
 	products = list(
+		/obj/item/clothing/suit/storage/marine/chemsuit = 10,
+		/obj/item/weapon/claymore/harvester = 15,
```

Comment:
I added it here because people I talked to on discord feel it is on the weak side and this would be a good place to add it for testing. Though if you like to I'll move it to req for 15 points or something like this. 

Also the question about using the suit+weapon requiring expending of a large quantity of meds to work was brought up, so req could spend points on that instead.